test_site/html/section_experiments.htm

```diff
@@ -51,24 +51,24 @@
       </div>
     </div>
 	
-	<div class='level-1'>
-		<div class='level-2'>
-			<div class='level-3'>
-				<div class='level-4'>
-					<div class='level-5'>
-						<span class='deep-span'>Deep span</span>
-					</div>
-					<div class='level-5'/>
-				</div>
-				<div class='level-4'>
-					<div class='level-5'>
-						<span class='deep-span'>Deep span</span>
-					</div>
-					<div class='level-5'/>
-				</div>
-			</div>
-		</div>
-	</div>
+    <div class='level-1'>
+      <div class='level-2'>
+        <div class='level-3'>
+          <div class='level-4'>
+            <div class='level-5'>
+              <span class='deep-span'>Deep span</span>
+            </div>
+            <div class='level-5'/>
```

Comment:
There's no doctype here, so if this works it's fine.  But I don't think a `div` tag is supposed to be self-closing.

Reply:
I'll add closures to all of these dives to fix them up (Where relevant)